src/ui/component/collapsible/collapsible.tsx

```diff
@@ -52,17 +60,18 @@ export const Collapsible: FC<CollapsibleProps> = ({
         <div
           className={classNames(
             'okp4-dataverse-portal-collapsible-trigger-icon',
-            {
-              flipped: openState
-            },
+            { flipped: openState },
             `${triggerClassName}-icon`
           )}
         >
           <Icon name={iconName} />
         </div>
       </RCollapsible.Trigger>
       <RCollapsible.Content
-        className={classNames('okp4-dataverse-portal-collapsible-content', contentClassName)}
+        className={classNames('okp4-dataverse-portal-collapsible-content', contentClassName, {
+          floating: floatingContent,
+          [`floating-${contentExpandDirection}`]: floatingContent
```

Comment:
I think here you could remove `floating: floatingContent,` and update this to :
```suggestion
          [`floating ${contentExpandDirection}`]: floatingContent
```

Reply:
That's a good trick to simplify conditional content class names. Commit has been updated according to your suggestion. Thanks!